MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+recursive-include openfisca_web_api/i18n *.mo
+include *.md
+include LICENSE
+include *.ini
```

Comment:
For the record: we're using [`setuptools` in `setup.py`](https://github.com/openfisca/openfisca-web-api/blob/d790243123c72fd198cc29a63d6dd4dedb316510/setup.py#L5)